manuscript.Rmd

```diff
@@ -424,30 +457,73 @@ In that case, the readers can be certain, that they have received a complete acc
+If the only goal of preregistration is to ensure confirmatory research, such changes are not justifiable.
+However, under our rational some changes may be justified.
+Any change would increase the uncertainty about the theoretical risk and may even decrease the theorethical risk.
+The changes still may be worthwhile, if the negative outcomes may be offset by an increase in detectability due to the change.
+Consider a preregistration that failed to specify what to do with missing values where the researchers subsequently encounter missings.
+In such case, the detectability is zero, since the data cannot be analyzed without a decision about how to handle the missing data.
+Any such decision would constitute a deviation from the preregistration.
+Furthermore, a reader cannot rule out that the researchers leveraged the decision to decrease theoretical risk, i.e., picking among all options the one that delivers the most beneficial results for the theory.
+Whatever decision they make, increased uncertainty about the theoretical risk is inevitable and the expected epistemic value is decreased compared to a world where they anticipated the need to deal with missing data.
+However, it is still justified to deviate.
+After all they have not anticipated the case and are left with a detectablilty of zero.
+Any decision will increase detectability to a non-zero value, likely offsetting the increase in uncertainty.
+The researchers also may do their best to argue that the deviation was not motivated by increasing theoretical risk, thereby, decreasing the uncertainty.
+Maybe there is one decision that fits well with the theory or with the study design.
+Or, if there is no obvious candidate, the researchers could conduct a multiverse analysis of the available options to deal with missings to show the influence of the decision [@steegenIncreasingTransparencyMultiverse2016].
+
+Using reduction in uncertainty as the objective for preregistration does not only explain some existing behavior, it allows to form recommendations to improve the practice of preregistration.
+Maybe most important, we now have a measure to differentiate a good preregistration from a bad one.
+In particular, a preregistration should be specific about the procedure that is intended to generate evidence for a theory.
+Such procedure may accommodate a wide range of possible data, i.e., it may be exploratory.
+The theoretical risk, however low, must be communicated clearly.
+Parts of the process left unspecified, imply uncertainty, which is what a preregistration should reduce.
+However, specifying procedures that can be expected to not work, will lead to deviation, and uncertainty later.
+
+We have proposed a highly specific, yet practical preregistration procedure called preregistration as code (PAC) elsewhere [@peikertReproducibleResearchTutorial2021].
+In a PAC, researchers use computer code instead of a verbal description.
```

Comment:
```suggestion
In PAC, there is no distinction between a preregistration document and a final scientific report because the final report including all data analyses is created as a dynamic document including dynamically generated statistical analyses and natural language parts.
```